common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -2162,7 +2162,8 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
         "Whether Hive enables the optimization about converting common join into mapjoin based on the input file size. \n" +
         "If this parameter is on, and the sum of size for n-1 of the tables/partitions for a n-way join is smaller than the\n" +
         "specified size, the join is directly converted to a mapjoin (there is no conditional task)."),
-
+    HIVE_CONVERT_ANTI_JOIN("hive.auto.convert.anti.join", false,
```

Comment:
@maheshk114 Have you run all the tests with this feature set to true by default? This change touches existing logic/code and we should definitely run all the existing tests with this set to TRUE.

Reply:
Yes, i had triggered a ptest run with this config enabled to true by default. There were some 26 failures. I had analyzed those and some fixes were done to make sure that the result is same for both and difference in plan is as expected.

Reply:
Is there any reason why we should not enable this by default in master? It seems it is always beneficial to execute the antijoin since we already have a vectorized implementation too. That would increase the test coverage for the feature.

Reply:
Agree with the above, I believe we should enable anti-join by default as 1) this feature should aways improve runtime 2) can help us find possible issues and further optimize existing implementation

Reply:
done